blockmanager.go

```diff
@@ -1063,7 +1073,7 @@ func (b *blockManager) handleInvMsg(imsg *invMsg) {
 			break
 		}
 	}
-	imsg.peer.requestQueue = requestQueue
+	imsg.peer.SetRequestQueue(requestQueue)
```

Comment:
Once the rest is changed, it'll likely affect this, so we can revisit then, but this doesn't feel right having something to muck with the internals of the peer.